packages/ui/src/components/skeletons/skeleton-list.tsx

```diff
@@ -1,61 +1,51 @@
-import { useMemo } from 'react'
+import { FC } from 'react'
 
-import { StackedList } from '@/components'
+import { Skeleton, StackedList } from '@/components'
 import { cn } from '@utils/cn'
 
-import { Skeleton } from './components/skeleton'
-
-// Helper function to generate random percentage width within a range
-const getRandomPercentageWidth = (min: number, max: number) => `${Math.floor(Math.random() * (max - min + 1)) + min}%`
-
-// Helper function to generate random pixel width within a range
-const getRandomPixelWidth = (min: number, max: number) => `${Math.floor(Math.random() * (max - min + 1)) + min}px`
-
-const listItems = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
-
-interface SkeletonListProps {
+export interface SkeletonListProps {
   className?: string
+  linesCount?: number
+  hasActions?: boolean
+  classNames?: {
+    item?: string
+    leftTitle?: string
+    leftDescription?: string
+    rightTitle?: string
+    rightDescription?: string
+    actions?: string
+  }
 }
 
-interface RandomWidths {
-  titleWidth: string
-  descriptionWidth: string
-  secondaryTitleWidth: string
-  secondaryDescriptionWidth: string
-}
-
-export const SkeletonList = ({ className }: SkeletonListProps) => {
-  // Calculate random widths only once on mount
-  const randomWidths = useMemo<RandomWidths[]>(() => {
-    return listItems.map(() => ({
-      titleWidth: getRandomPercentageWidth(20, 60),
-      descriptionWidth: getRandomPercentageWidth(30, 80),
-      secondaryTitleWidth: getRandomPixelWidth(80, 150),
-      secondaryDescriptionWidth: getRandomPixelWidth(150, 250)
-    }))
-  }, []) // Empty dependency array ensures this runs only once on mount
-
+export const SkeletonList: FC<SkeletonListProps> = ({ classNames = {}, linesCount = 8, hasActions, className }) => {
+  const { item, leftTitle, leftDescription, rightTitle, rightDescription, actions } = classNames
   return (
-    <div className={cn('relative h-full w-full transition-opacity delay-500 duration-500 ease-in-out', className)}>
+    <div className={cn('cn-skeleton-list', className)}>
```

Comment:
Can we move this classname to the StackedList.Root and get rid of extra wrapper?